src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figures {
+    private int leftLeg;
+    private int rightLeg;
+
+    public RightTriangle(String color, int leftLeg, int rightLeg) {
+        super(color);
+        this.leftLeg = leftLeg;
+        this.rightLeg = rightLeg;
+    }
+
+    @Override
+    public double area() {

```

Comment:
The method name `area` is not following Java naming conventions for methods. Methods should be verbs, in mixed case with the first letter lowercase and with the first letter of each internal word capitalized. Consider renaming it to `calculateArea`.